ng-dev/index.ts

```diff
@@ -22,6 +22,12 @@ export * from './release/versioning';
 // to the dev-infra log which is stored on failures.
 export * from './utils/console';
 
+// Exposes Git/Github client classes needed for interacting with some of the
+// release versioning APIs.
+export * from './utils/git/authenticated-git-client';
```

Comment:
Can change this to expose the specific symbols we want rather than all?

Reply:
Yeah, could do. Just not sure if it's worth maintaining that list. It's already quite unclear what specific APIs e.g. the versioning API transitively requires 😓

Reply:
Let me know, happy to go with either approach (star would be consistent with the file curently)

Reply:
We can leave it as is, I am surprised we aren't able to just provide the specific ones here as for instance `./utils/git/authenticated-git-client` only has two exported symbols.